### ISSUE TITLE
Flycast crashing on closing content

### ISSUE THREAD

**Issue by klepp0906** [libretro/flycast#1057]
It simply closes retroarch entirely if I load a game then use the quick menu to "close content" as opposed to returning to the retroarch menu.

tried with glcore and with vulkan.  both produced the same result.

windows 10

**Comment by afjara** [comment]
Yes, It has been a problem especially for some nvidia gpus and the builds released a few months ago, still not resolved. #974
